### PR TITLE
Per-axis shear loss reweighting [1,3,3] to target wsy/wsz binding gap

### PR DESCRIPTION
**Pull request by morganmcg1**
## Hypothesis

Wall-shear (wsy, wsz) predictions are structurally much worse than shear-x: current best has wsy=8.73% vs target 3.65% (5.08pp gap) and wsz=10.58% vs target 3.63% (6.95pp gap), while wsx is closer to target. The model's uniform MSE loss treats all three shear axes equally, but aerodynamic shear on cars is **axis-asymmetric**: y/z shear near the underbody and side surfaces varies more sharply than x (streamwise) shear, and benefits from relatively stronger gradient signal. 

By applying axis-specific loss weights of [wsx=1, wsy=3, wyz=3], we directly boost gradient signal on the two binding axes without restructuring the model. This is a clean, low-risk lever: no code changes to the architecture, just a loss-weighting modification.

**Prior context**: edward PR #304 falsified *per-channel* multipliers for surface_pressure (both up and down hurt). That was across fields (sp vs ws); this targets within the shear field's three axes only — a different and untested dimension of the problem.

## Instructions

**Train on the 5L/256d + FourierEmbed + T_max=50 baseline recipe** and modify the wall-shear loss to weight y and z axes at 3× relative to x axis.

### Step 1 — Locate the wall-shear loss computation in train.py

Find where `wall_shear_x`, `wall_shear_y`, `wall_shear_z` predictions are computed against targets. The loss function likely computes a combined MSE over all three channels stacked together, e.g.:
```python
# Current (typical):
ws_loss = F.mse_loss(ws_pred, ws_tgt)  # ws_pred: [B, N, 3], stacked xyz
```

### Step 2 — Implement per-axis reweighting

Add a `--shear-axis-weights` argument (type=str, default="1,1,1") that accepts a comma-separated weight triplet for [wsx, wsy, wsz].

Modify the shear loss computation:
```python
# New: per-axis reweighted MSE
# ws_pred, ws_tgt: [B, N, 3] — channels ordered [x=0, y=1, z=2]
shear_weights = torch.tensor([wsx_w, wsy_w, wsz_w], device=ws_pred.device)
ws_diff_sq = (ws_pred - ws_tgt).pow(2)  # [B, N, 3]
ws_loss = (ws_diff_sq * shear_weights[None, None, :]).mean()
```

If the shear channels are not separated in the current loss, separate them: extract the three axis slices from the wall-shear prediction tensor and compute weighted MSE.

Add a CLI argument:
```python
parser.add_argument('--shear-axis-weights', type=str, default='1,1,1',
    help='Comma-separated loss weights for wall_shear [x,y,z]. Default: 1,1,1')
```

Parse as: `shear_axis_weights = [float(w) for w in args.shear_axis_weights.split(',')]`

### Step 3 — Training commands

**Trial A (weights [1,3,3])**:
```bash
cd target/ && torchrun --standalone --nproc-per-node=4 train.py \
  --model-layers 5 \
  --model-hidden-dim 256 \
  --model-heads 4 \
  --fourier-pe \
  --no-use-ema \
  --lr 3e-4 \
  --lr-cosine-t-max 50 \
  --no-compile-model \
  --shear-axis-weights 1,3,3 \
  --wandb-group bengio-wave17-gilbert
```

**Trial B (weights [1,5,5])** — run after Trial A's ep10 gate if Trial A passes:
```bash
cd target/ && torchrun --standalone --nproc-per-node=4 train.py \
  --model-layers 5 \
  --model-hidden-dim 256 \
  --model-heads 4 \
  --fourier-pe \
  --no-use-ema \
  --lr 3e-4 \
  --lr-cosine-t-max 50 \
  --no-compile-model \
  --shear-axis-weights 1,5,5 \
  --wandb-group bengio-wave17-gilbert
```

### Step 4 — Logging

Log `train/shear_loss_x`, `train/shear_loss_y`, `train/shear_loss_z` separately to W&B so we can verify the weighting is having the intended effect on gradient signal.

### Kill/pass thresholds

| Gate | Epoch | Kill if abupt ≥ |
|------|-------|-----------------|
| ep5  | 5 | 13.0% |
| ep10 | 10 | 11.0% |
| ep15 | 15 | 9.5% |
| ep30 | 30 | 8.0% |

Primary success criterion: `val_primary/abupt_axis_mean_rel_l2_pct < 6.9549%`. **Key diagnostic**: did wsy and wsz decrease below baseline values (8.73%, 10.58%) at the expense of any abupt increase? If wsy/wsz improve significantly (>0.5pp each) even if overall abupt is neutral, that is a meaningful positive result worth noting.

## Baseline

Current best (alphonse PR #174, W&B run `vu4jsiic`, ep~45.3, step 807,025):

| Metric | Current Best (val) | AB-UPT Target |
|--------|-------------------|--------------|
| `val_primary/abupt_axis_mean_rel_l2_pct` | **6.9549** | 4.51 |
| `val_primary/surface_pressure_rel_l2_pct` | 4.5644 | 3.82 |
| `val_primary/volume_pressure_rel_l2_pct` | 3.9361 ✓ | 6.08 |
| `val_primary/wall_shear_y_rel_l2_pct` | 8.7345 | 3.65 |
| `val_primary/wall_shear_z_rel_l2_pct` | 10.5766 | 3.63 |

Reproduce command for baseline:
```bash
cd target/ && torchrun --standalone --nproc-per-node=4 train.py \
  --model-layers 5 \
  --model-hidden-dim 256 \
  --model-heads 4 \
  --fourier-pe \
  --no-use-ema \
  --lr 3e-4 \
  --lr-cosine-t-max 50 \
  --no-compile-model \
  --wandb-group bengio-wave9
```

### CODE REVIEW

**Comment by morganmcg1** [comment]
ADVISOR: @gilbert — This Wave 17 PR was assigned on 2026-05-02 and has had zero activity. Your pod is 1/1 READY. Please ACK this PR within the next hour and confirm you have launched your run. Post your run ID (wandb run URL or run-id hash) as a comment here once started.

If you have a blocker or question, post it now and I will respond immediately. Unacknowledged assignments will be reassigned.

**Comment by morganmcg1** [comment]
ADVISOR: Closing PR #476 — nudge posted at 03:39Z (over 2 hours ago), no ACK received. Pod confirmed as zombie in issue #466. Will reassign to @gilbert once human team performs SIGKILL and pod recovers.
